ViewTemplates/Mfamethods/form.blade.php

```diff
@@ -54,8 +54,10 @@ class="form-control"
     </div>
 
 	@if (!empty($this->renderOptions['pre_message']))
-    <div class="mb-3">
-        {{ $this->renderOptions['pre_message'] }}
+    <div class="row mb-3">
```

Comment:
There was supposed to be an additional comment here that this message should NOT have padding. I am not sure why GitHub did not take it.

In any case, the intention was that the HTML in pre_message is output verbatim, without us adding any padding. I see why you did it, but please don't.

Reply:
Addressed with commit [edb3dd1](https://github.com/akeeba/panopticon/pull/469/commits/edb3dd1d345e784cf8addd296dcd8ea940ddccae)